app/models/cms/builtin/legal_term.rb

```diff
@@ -3,10 +3,12 @@ class CMS::Builtin::LegalTerm < CMS::Builtin::Partial
   SIGNUP_SYSTEM_NAME = 'signup_licence'
   SUBSCRIPTION_SYSTEM_NAME = 'service_subscription_licence'
   NEW_APPLICATION_SYSTEM_NAME = 'new_application_licence'
-  
+
   validates :published, presence: true
   validates :title, uniqueness: { :scope => [:provider_id], case_sensitive: true }
 
+  before_validation :publish_draft
```

Comment:
Wouldn't it be better `before_save`? If I'm not mistaken, the draft will be published regardless of the template's validity.

Reply:
The problem is that `before_save` runs **after** the validation. And if we don't fill `published` with a value before validation, the validation will fail :grimacing: